content/fast/defer-unused-css/index.md

```diff
@@ -0,0 +1,149 @@
+```
+
+This is not the standard way of loading CSS. Here's how it works:
+
+* ```link rel=”preload” as=”style”``` allows to request the style asynchronously (for more information about ```"preload"```  , check out [this guide](https://web.dev/fast/preload-critical-assets)).
+* The ```onload``` attribute on the link allows the CSS to be processed, when it finishes loading.
+* “nulling” the ```onload``` handler once it is used, helps some browsers to avoid re-calling the handler upon switching the rel attribute.
+* The ordinary reference to the stylesheet inside a noscript element works as a fallback, for browsers that don’t execute JS.
+
+The [resulting page](https://defer-css-optimized.glitch.me/) looks exactly like the previous version, even when most of its styles are loaded asynchronously:
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    allow="geolocation; microphone; camera; midi; encrypted-media"
+    src="https://glitch.com/embed/#!/embed/defer-css-optimized?path=index.html&previewSize=100&attributionHidden=true"
+    alt="defer-css-optimized on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+## Monitor
+
+Obtain a new Performance trace with DevTools, now running it on the [optimized page](https://defer-css-optimized.glitch.me/). 
+
+The **FCP** marker appears before the page requests the CSS, which means the browser doesn’t need to wait for the CSS to be loaded to start rendering the page:
+
+<figure>
+  <img src="./cdt-perf-optimized.png"
+       alt="DevTools performance trace for unoptimized page, showing FCP starting before CSS loads." class="screenshot">
+</figure>
+
+As a final step, run Lighthouse on the optimized page. 
+
+On the report you’ll see that the FCP page has been reduced in **0.2s** (a 20% improvement!):
+
+<figure>
+  <img src="./lighthouse-metrics-optimized.png"
+       alt="Lighthouse report, showing an FCP value of '1s'." class="screenshot">
+</figure>
+
+Also, as a result, the **Eliminate render-blocking resources** suggestion is no longer under **Opportunities**, and now belongs to the **Passed Audits** section:
+
+<figure>
+  <img src="./lighthouse-opportunities-optimized.png"
+       alt="Lighthouse report, showing 'Eliminate blocing resources' on the 'Passed Audits' section." class="screenshot">
+</figure>
+
+## Next steps & references
+
+On this guide you used vanilla code to implement this optimization. In a real production scenario, it’s a good practice use functions like [loadCSS](https://github.com/filamentgroup/loadCSS/blob/master/README.md), that can encapsulate this behavior and work well across browsers. As a complement of this, you can use tools like [Critical](https://github.com/addyosmani/critical/blob/master/README.md), which helps extracting and inlining the “Above the Fold” CSS, you did manually on the **Optimize** step.
+These tools will be covered in an upcoming guide.
```

Comment:
```suggestion

```

I'd suggest removing this bit because I have a hunch we'll forget to remove it when we publish the other guides.